target/linux/ramips/mt7620/base-files/etc/board.d/02_network

```diff
@@ -217,6 +217,10 @@ ramips_setup_interfaces()
 		ucidef_add_switch "switch0" \
 			"0:lan" "4:wan" "6@eth0"
 		;;
+	wavlink,wl-wn579x3)
+		ucidef_add_switch "switch0" \
+			"0:lan" "1:lan" "2:lan" "3:lan" "5:lan" "4:wan" "6@eth0"
```

Comment:
Isn't it a two-port device? Why 5 LAN ports?

Reply:
Good point, I've just gone by the output of `swconfig show` but you're right only ports 4/5 are used.